hashing/te-tag-query-python/all_in_one_lib/signal_type/url.py

```diff
@@ -0,0 +1,24 @@
+##!/usr/bin/env python
+
+"""
+Wrapper around the URL content type.
```

Comment:
maybe some comments about how URL normalization is handled -- ?

Reply:
Currently not handled :'( this was a victim of picking an arbitrary cutoff to try and get some things into the repo.

There's some python libs for URL normalization that look good for the long term